.github/workflows/run_tests.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.5.x, 2.6.x]
-        appraisal: ["rails-4", "rails-5", "rails-6"]
+        ruby: [2.7.x]
+        appraisal: ["rails-5", "rails-6"]
```

Comment:
**Note**: Making changes to this file to get past CI

First CI run: https://github.com/Gusto/ar-query-matchers/actions/runs/854049577
Second CI run: https://github.com/Gusto/ar-query-matchers/pull/25/checks?check_run_id=2615637605

It seems some dependencies no longer work for ruby 2.5.x and rails-4